### PR TITLE
port localization to stable

### PR DESCRIPTION
**Pull request by colombod**

### CODE REVIEW

File: src/polyglot-notebooks-vscode/do-version-upgrade.ps1 [hunk 1]
```diff
@@ -15,14 +15,14 @@ try {
     & node .\tools\buildSemanticTokenScopes.js
     Pop-Location
 
-    # copy package.json
-    Copy-Item -Path "$stableDirectory\package.json" -Destination "$insidersDirectory\package.json"
+    # copy package.json from insider
```
Comment (colombod) [inline src/polyglot-notebooks-vscode/do-version-upgrade.ps1:18]: will copy insider package.json so we have all localization set properly

File: src/polyglot-notebooks-vscode/do-version-upgrade.ps1 [hunk 2]
```diff
@@ -15,14 +15,14 @@ try {
...
+    Copy-Item -Path "$insidersDirectory\package.json" -Destination  "$stableDirectory\package.json"
 
-    $insidersPackageJsonContents = (Get-Content "$insidersDirectory\package.json" | Out-String | ConvertFrom-Json)
+    $stablePackageJsonContents = (Get-Content "$stableDirectory\package.json" | Out-String | ConvertFrom-Json)
 
-    $insidersPackageJsonContents.scripts.package += " --pre-release" 
+    $stablePackageJsonContents.scripts.package = $stablePackageJsonContents.scripts.package.Replace("--pre-release","").Trim()
```
Comment (colombod) [inline src/polyglot-notebooks-vscode/do-version-upgrade.ps1:23]: make sure to remove the `--pre-release` option

File: src/polyglot-notebooks-vscode/do-version-upgrade.ps1 [hunk 3]
```diff
@@ -36,6 +36,12 @@ try {
     Remove-Item -Path "$stableDirectory\src\*" -Filter "*.ts"
     Copy-Item -Path "$insidersDirectory\src\*" -Destination "$stableDirectory\src\" -Filter "*.ts"
 
+    # copy localization files
```
Comment (AbhitejJohn) [inline src/polyglot-notebooks-vscode/do-version-upgrade.ps1:39]: Quick question: So if we have this in the script, do we still need the package.nls.*.json files below? Looks like they exist here: https://github.com/dotnet/interactive/tree/main/src/polyglot-notebooks-vscode-insiders

**Review by ocallesp — APPROVED** [review]

**Comment by ocallesp** [comment]
Polyglot-notebooks-vscode
image

File: src/polyglot-notebooks-vscode/do-version-upgrade.ps1 (hunk 3 above)
Comment (ocallesp) [reply src/polyglot-notebooks-vscode/do-version-upgrade.ps1:39]: package.nls.*.json should override the existing ones to keep in sync with the ones in insiders

File: src/polyglot-notebooks-vscode-insiders/package.json [hunk 4]
```diff
@@ -841,6 +841,6 @@
     "vscode-oniguruma": "1.6.1",
     "vscode-textmate": "6.0.0",
     "vscode-uri": "3.0.6",
-    "@vscode/l10n": "^0.0.10"
+    "@vscode/l10n": "0.0.10"
   }
 }
```
[the remark this answers is not in the document]
Comment (colombod) [reply src/polyglot-notebooks-vscode-insiders/package.json:846]: the release branch has that change already

**Pull request merged**